tests/dynamic_batch_sizes.rs

```diff
@@ -53,6 +53,8 @@ async fn dynamic_batch_sizes() -> anyhow::Result<()> {
         "10",
         "--tree-gc-threshold",
         "1",
+        "--dense-tree-mmap-file",
+        "./testfile",
```

Comment:
This will breat acceptance tests - they're treated as separate binaries and run concurrently. Can you get a temporary file path using https://crates.io/crates/tempfile ?

Reply:
Wouldn't `#[serial_test::serial]` prevent that? That crate seems cool, i'll see what i can do

Reply:
I think you'd have to use https://docs.rs/serial_test/latest/serial_test/attr.file_serial.html, but also I'd like these tests to run in parallel - they quite long so we could be looking at a difference between 30-45 seconds to run the whole test stuie to like 1m 30s.

btw. this is technically undefined behaviour - since multiple processes will try to mutably map the same file to memory

Reply:
Couldn't use temp file, but instead used temp directory. It creates a dir in /tmp and then testfile is made inside. 
On drop it will delete the directory, so i think its fine solution. Also all directories are differently named (random gen).